tensorflow_addons/image/sparse_image_warp_test.py

```diff
@@ -0,0 +1,255 @@
+# Copyright 2018 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Tests for sparse_image_warp."""
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import numpy as np
+import tensorflow as tf
+import tensorflow.compat.v1 as tf1  # TODO: port TF1 test files?
+from tensorflow_addons.image import _get_boundary_locations
```

Comment:
Hi Kyle, thanks for all your great work! Do you mind replacing these with for example:
`from tensorflow_addons.image.sparse_image_warp import _get_boundary_locations`

It's probably best we keep the underscore prefix methods out of the __init__

Reply:
done